src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -789,6 +791,8 @@ public String getCsvDetailedResponsesRow(FeedbackSessionResultsBundle fsrBundle,
                     + SanitizationHelper.sanitizeForCsv(chosenIndexString) + ','
                     + SanitizationHelper.sanitizeForCsv(chosenChoiceValue) + ','
                     + SanitizationHelper.sanitizeForCsv(chosenChoiceNumber)
+                    + (hasCommentsForResponses
+                        ? fsrBundle.getCsvDetailedFeedbackResponseCommentsRow(feedbackResponseAttributes) : "")
```

Comment:
Might have missed this: this should use 8 spaces. If doing so would exceed the line length limit, putting the `: "")` part to the next line could work.